firmware/user/display/oled.c

```diff
@@ -202,6 +202,198 @@ void drawPixel(int16_t x, int16_t y, color c)
     }
 }
 
+
+/**
+ * Set/clear/invert a single pixel unsafely but quickly.
+ *
+ * This intentionally does not have ICACHE_FLASH_ATTR because it may be called often
+ *
+ * @param x Column of display, 0 is at the left
+ * @param y Row of the display, 0 is at the top
+ */
+
+void drawPixelUnsafe( int x, int y )
+{
+    uint8_t* addy = &currentFb[(y + x * OLED_HEIGHT) / 8];
```

Comment:
Would `currentFb[(y + x * OLED_HEIGHT) / 8] |= (1 << (y & 7));` be any faster? I would think not, but you never know with this compiler...

Reply:
I originally had that.  This is exactly the same speed but takes less memory.